src/common/config/RegistryConfig.js

```diff
@@ -3,11 +3,20 @@
 // See LICENSE.txt for license information.
 
 import {EventEmitter} from 'events';
+import log from 'electron-log';
 
-import WindowsRegistry from 'winreg';
+let registry;
+try {
+  // eslint-disable-next-line global-require
+  registry = require('registry-js');
+} catch (e) {
+  if (process.platform === 'win32') {
```

Comment:
How come the check for `win32` is only performed here?

Reply:
there is no registry for win32, so only if we are running win32 will show the error. I don't expect it to work on mac or linux.